components/tabcordion/src/Tab.js

```diff
@@ -135,7 +137,7 @@ export const Tab = forwardRef(
 		useEffect(() => {
 			if (mode === 'tabs') {
 				if (selected) {
-					onOpen(tabId);
+					onOpen(idx);
 				} else {
 					onClose(tabId);
 				}
```

Comment:
We should probably pass both idx and tabId to all of the callback functions for consistency

Reply:
yeah let's do both

Reply:
I'd do that as an object and favor `idx` over `tabId` as it's more specific.